tssc/step_implementers/shared/openscap_generic.py

```diff
@@ -0,0 +1,656 @@
+"""Step Implementer for the container-image-static-compliance-scan step for OpenSCAP.
+
+Step Configuration
+------------------
+
+Step configuration expected as input to this step.
+Could come from either configuration file or
+from runtime configuration.
+
+| Configuration Key              | Required? | Default | Description
+|--------------------------------|-----------|---------|-----------
+| `oscap-input-definitions-uri`  | Yes       |         | URI to the OpenSCAP definitions file
+|                                |           |         | to do the evaluation with.
+|                                |           |         | Must use protocol file://|http://|https://.
+|                                |           |         | Must have file extension .xml|.bz2.
+
+| `oscap-profile`                | No        |         | OpenSCAP profile to evaluate.
+
+| `oscap-tailoring-uri`          | No        |         | URI to OpenSCAP tailoring file
+|                                |           |         | to do the evaluation with.
+|                                |           |         | Must use protocol file://|http://|https://.
+|                                |           |         | Must have file extension .xml|.bz2.
+
```

Comment:
These newlines are superfluous

Reply:
it was impossible for me to read without them.

Reply:
Fair. Maybe I've gotten used to dense blocks thanks to a lot of C in the last year.

Reply:
this actually broke the p3doc rendering after testing. fixed.